Exercise_B/index.css

```diff
@@ -1,4 +1,22 @@
-/* 
-This css sheet is linked to index.html
-You can style tags, classes (.) or ids (#) from this file.
-*/
+.box1 {
+background-color: blue;
+height: 100px;
+width: 100px;
+border: 1px solid black;
+float: left;
+}
+.box2 {
+background-color: green;
+height: 100px;
+width: 100px;
+border: 1px solid black;
+float: right;
+}
+.box3 {
+background-color: red;
+height: 100px;
+width: 100px;
+border: 1px solid black;
+position: inherit;
+bottom: 300px;
+}
```

Comment:
For this, it seems to be that your .box3 was hidden. When using position, you can use clearfix or there is also position absolute. You can try this 
```
.box3 {
  background-color: red;
  height: 100px;
  width: 100px;
  border: 1px solid black;
  position: absolute;
  bottom:500px;
  left:750px;
}
```